shared/src/main/scala/effekt/Parser.scala

```diff
@@ -238,7 +238,7 @@ class Parser(positions: Positions) extends Parsers(positions) with Phase[Source,
   lazy val effectDef: P[Def] =
     ( `effect` ~> effectOp ^^ {
         case op =>
-          EffDef(IdRef(op.id.name) withPositionOf op.id, List(op))
+          EffDef(IdDef(op.id.name) withPositionOf op.id, List(op))
```

Comment:
Not sure if this change will break something. If yes this pull request is obsolete.